site/en/guide/checkpoint.ipynb

```diff
@@ -139,7 +139,7 @@
         "## Saving from `tf.keras` training APIs\n",
         "\n",
         "See the [`tf.keras` guide on saving and\n",
-        "restoring](./keras/overview.ipynb#save_and_restore).\n",
+        "restoring](saved_model.ipynb).\n",
```

Comment:
We can also use a full link for _Keras-only guides_ (source https://github.com/keras-team/keras-io/blob/master/guides/serialization_and_saving.py):

```suggestion
        "restoring](https://www.tensorflow.org/guide/keras/save_and_serialize).\n",
```

as per @MarkDaoust 's suggestion